README.md

```diff
@@ -2,208 +2,394 @@
+As we saw before, `favoriteFruit` does not get scoped to the `if` statement's
+block. So, when we try to access the variable inside the `logAnimalFruit`
+function that it's nested within, it is available. However, when we try to
+access it _outside_ of the function like we did before, we are now unable to
+access it.
+
+In sum, `var` gets scoped to its nearest parent function. If it's not declared
+within a function at all, it gets scoped globally.
+
+Confusing, isn't it? This is all the more reason to **never use `var`** when
+declaring variables. As long as you stick to declaring variables with `const`
+and `let`, what happens in block stays in block.
+
+## The Missing Keyword Gotcha
+
+If you declare a variable with no keyword whatsoever, that variable **is always
+globally scoped**. Regardless of where you define this variable, it will always
+be accessible anywhere after its declaration.
+
+For example, if we declare a variable with no keyword inside a function, it is
+still available anywhere:
 
 ```js
 function bankAccount() {
-  secretPassword = "il0v3pupp135";
-
-  return "bankAccount() function invoked!";
+  username = "corgiluvr";
+  secretPassword = "il0v3pupp!35";
+  money = 1000000000;
+  console.log("Account created for: " + username); // => "Account created for: corgiluvr"
 }
 
 bankAccount();
-// => "bankAccount() function invoked!"
+console.log(secretPassword); // => "il0v3pupp!35"
 
-secretPassword;
-// => "il0v3pupp135"
+if ((secretPassword = "il0v3pupp!35")) {
+  const withdraw = 1000000000;
+  money = money - withdraw;
+  console.log("Withdrew amount: " + withdraw); // => Withdrew amount: 1000000000
+  console.log("Remaining balance: " + money); // => Remaining balance: 0
+}
 ```
 
 Oh no; our super secret password has leaked into the global scope and is
-available everywhere! Declaring global variables and functions should only be
-used as a last resort if you absolutely need access to something **everywhere**
-in your program. In general, it's best practice to make variables and functions
-available only where they're needed — and nowhere else. Making a variable
-available in places that shouldn't have access to it can only lead to bad things
-and make your debugging process more complex. The more pieces of code that can
-access a given variable, the more places you have to check for bugs if/when that
-variable contains an unexpected value.
+available everywhere! A bad actor exploited that and created a simple `if`
+statement with our password to drain all our money - which was also available
+everywhere.
 
-## Conclusion
+Declaring global variables and functions should only be used as a last resort if
+you absolutely need access to something **everywhere** in your program.
+
+Again, it is best practice to always declare variables with either `let` or
+`const`. Doing so will avoid troubling issues like these!
+
+## Practice Identifying Scope
+
+We've learned a lot! I'm sure you can see now why scope is a tricky beast that
```

Comment:
```suggestion
We've learned a lot! 
```

Reply:
I think we can avoid overemphasizing the difficulty of the topic while still encouraging students that it's ok if they don't fully understand it.